.travis.yml

```diff
@@ -80,7 +80,7 @@ install:
   - phing recall -Dversion=$VERSION -Ddb.database=drupal -Ddb.user=lightning -Ddb.password=lightning -Ddb.host=127.0.0.1
   - cd docroot
   # Execute database updates, if there are any.
-  - drush updatedb --yes
+  - drupal update:execute
```

Comment:
It is necessary to get the tests to pass, but can we update the documentation in another PR?